### PR TITLE
[Feature:Forum] display sum duck for thread

### PR DESCRIPTION
**Pull request by yanliw123**
fix https://github.com/Submitty/Submitty/issues/10732
### What is the new behavior?
imageDisplay sum duck for threads 
duck icon on bottom-left corner, keep consistency with post duck icon
may help student find more valuable threads
### Other information?

log into different accounts, like the thread see if the sum duck is correct

### CODE REVIEW

**Comment by ZheyuDeng** [comment]
please link the issue

**Comment by yanliw123** [comment]
> please link the issue

https://github.com/Submitty/Submitty/issues/10732
>.<

**Comment by williamschen23** [comment]
you need to put fix #10732  to close the issue as done

File: site/cypress/e2e/Cypress-UI/pronouns_change.spec.js
Comment (IDzyre) [inline site/cypress/e2e/Cypress-UI/pronouns_change.spec.js:80]: We should eventually refactor at least some of this file to use data-testids. Either you can add them here instead of using html selectors (.xxx or #xxx), or create an issue and have it changed in the near future. I don't usually check out the branch to try the test on the first round of reviews, and I just read through the code to see if there is anything that should be done differently (best practice, or an easier way to do something, etc), and it is difficult to do that on this file as it was created before we started emphasizing testids.

**Review by zacharymnp — APPROVED** [review]
Found no issues in code or testing.

**Review by williamjallen — CHANGES_REQUESTED** [review]
I left a few minor nitpick comments, but this looks pretty good overall.

Can you please add a Cypress test which specifically tests this new functionality though?  Every new feature should ideally be tested via Cypress.

I would also like @DarthNyan to take a look at this and decide whether it would be better to merge this before or after the Doctrine refactor is merged.

**Review by DarthNyan — COMMENTED** [review]
I think these changes can go ahead now if we can get them merged soon. It'll be very simple to get this compatible with the Doctrine refactor, so I see no reason to hold this up.

**Review by williamjallen — CHANGES_REQUESTED** [review]
The code looks good to me overall, but I'd still like to see a Cypress test for this feature.  Every new UI feature should have at least a minimal Cypress test.

**Comment by yanliw123** [comment]
> The code looks good to me overall, but I'd still like to see a Cypress test for this feature. Every new UI feature should have at least a minimal Cypress test.

I put the cypress test for sum thread duck into a seperate PR #11012. Because I have a refactor cypress test PR#11003. There will gonna be version conflicts, so I put them into seperate pr, and will fix the version conflicts after this PR and PR #11003 get merged.

**Pull request merged**
